crates/ruma-state-res/Cargo.toml

```diff
@@ -15,7 +15,11 @@ edition = "2018"
 all-features = true
 
 [features]
-unstable-pre-spec = ["ruma-events/unstable-pre-spec"]
+unstable-pre-spec = ["ruma-common/unstable-pre-spec", "ruma-events/unstable-pre-spec"]
```

Comment:
Is the dependency on `ruma-common/unstable-pre-spec` actually needed? I.e. does `ruma-state-res` use unstable things from ruma-common when compiled with `unstable-pre-spec` itself?